<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 71, 127);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>File</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeft|AlignVCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diagram</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grid</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="filePage"/>
       <widget class="QWidget" name="diagramPage">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="toolsWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout" rowstretch="0">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="2">
             <widget class="QToolButton" name="nodeToolButton">
              <property name="text">
               <string>N</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="nodeLinkToolButton">
              <property name="text">
               <string>L</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3" alignment="Qt::AlignLeft">
             <widget class="QToolButton" name="moveToolButton">
              <property name="text">
               <string>M</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="OptionsStackedWidget" name="optionsWidget">
           <widget class="QWidget" name="selectionOptions"/>
           <widget class="QWidget" name="nodeOptions">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="NodeTypeListWidget" name="nodeTypeListWidget">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridPage">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="gridCheckBox">
           <property name="text">
            <string>Grid</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="DiagramView" name="diagramView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DiagramView</class>
   <extends>QGraphicsView</extends>
   <header>diagram/diagramview.h</header>
  </customwidget>
  <customwidget>
   <class>NodeTypeListWidget</class>
   <extends>QListWidget</extends>
   <header>nodetype/nodetypelistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>OptionsStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>nodetype/optionsstackedwidget.h</header>
   <container>1</container>
   <slots>
    <slot>showSelectionOptions()</slot>
    <slot>showNodeOptions()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
